### PR TITLE
Improve docs of author and post metadata

### LINKED ISSUE #208: Add use of preface YAML option for cross posts and multiple language posts
- [ ] Add description to [YAML instructions](https://blogguide.ropensci.org/authortechnical.html#whatgoesinyaml) for use of the preface option when crossposting or creating multiple language posts. 
- [ ] Add check in Editor lists to make sure it's there if necessary

### LINKED ISSUE #209: Adding how to create author profile
To add on: https://blogguide.ropensci.org/authortemplate.html

### Author
* Create a folder with the name of the author on content/author
* Copy a picture for the profile in themes/ropensci/static/img/community unless they have a GitHub profile with a picture, which will be used by default.
* Create an _index.md with the following content:

```markdown
---
name: Name Of The Author
github: github user
link: web page link
img: img/community/name-of-the-author-picture.png
---
```

### PR DESCRIPTION
**Pull request by maelle**
Fix https://github.com/ropensci/roweb3/issues/443
Fix #208
Fix #209

### CODE REVIEW

**Review by yabellini — APPROVED** [review]
Thanks, @maelle looks perfect. :-)  Do we have the permission of the person in the example to use their author profile on the guide?

**Comment by maelle** [comment]
oops no good point. maybe we could use a newer one as we're more in contact with them?

File: inst/checklists/editor-checklist.csv [hunk 1]
```diff
@@ -10,3 +10,4 @@ I ran `roblog::ro_lint_md()` on index.md
 I ran `roblog::ro_check_urls()` on index.md
 I ran a spell-check on index.md
 YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
+preface YAML field is present if necessary
```
Comment (steffilazerte) [inline inst/checklists/editor-checklist.csv:13]: Just to make it a bit more parallel to the above point
```suggestion
YAML field 'preface' is present if necessary
```

**Comment by steffilazerte** [comment]
Looks like the preview failed (I reran to be sure it wasn't a glitch). Is that something we need to fix too? Other than a quick look at the rendered book (just to make sure), and my one comment above, looks great, thanks!

**Comment by maelle** [comment]
@yabellini any suggestion of author file we could use?

**Comment by yabellini** [comment]
Perhaps Pao ?, Ale and Lucio's profiles don't have many data.

File: inst/templates/author-file-template.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
 ---
 name: {author_name}
```
Comment (steffilazerte) [inline inst/templates/author-file-template.md:2]: Why are some with {} and some not?

File: inst/templates/author-file-template.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 link: website URL or other online presence
+mastodon: Mastodon full URL to profile
 twitter: Twitter username
 github: {github_username}
 gitlab: GitLab username
 keybase: Keybase ID
 orcid: {orcid_id}
+img: link to picture to be saved in themes/ropensci/static/img/community
```
Comment (steffilazerte) [inline inst/templates/author-file-template.md:10]:
```suggestion
img: link to picture to be saved in themes/ropensci/static/img/community (if blank, uses GitHub profile picture)
```

File: inst/templates/author-file-template.md (hunk 2 above)
Comment (maelle) [reply inst/templates/author-file-template.md:2]: a mistake!

**Pull request merged**